.eslintrc.cjs

```diff
@@ -3,6 +3,9 @@ require("@rushstack/eslint-patch/modern-module-resolution");
 
 module.exports = {
   "root": true,
+  "rules": {
+    "vue/multi-word-component-names": 0
```

Comment:
isn't it best practice to use multi word component names in vue? Why is it turned off?

Reply:
I see that VueForge has added component with one word name but they are basically inside pages directory. Maybe rather than turning of multi word component linter for all project, we can just skip linter inside pages directory

Reply:
The generic rule was removed and add just for specific files